utilities/copyoffload_migration.py

```
@@ -407,6 +409,96 @@ def _find_populate_pods(ocp_admin_client: DynamicClient, namespace: str, migrati
+        populate_pods: list[Pod] = [
+            pod
+            for pod in Pod.get(
+                client=ocp_admin_client,
+                namespace=namespace,
+                label_selector=f"migration={migration_uid}",
+            )
+            if pod.name.startswith("populate-")
+        ]
+
+        if not populate_pods:
+            LOGGER.debug(f"No populate pods found for migration '{migration_uid}' (non-copyoffload or not yet started)")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)